perun-base/src/main/resources/logback-default.xml

```diff
@@ -0,0 +1,317 @@
+<!--
+	This is the default config for the Logback logging framework in Perun.
+	It is inherited by transitive dependencies into all Perun modules.
+	
+    You can load another file by e.g.
+     -Dlogback.configurationFile=file:///etc/perun/logback.xml
+    and change the directory for log files by e.g.
+      -Dperun.log=/var/log/perun
```

Comment:
I believe there should be slash at the end ``/var/log/perun/`` or it might not work (it's like that in config and pom.xml).